flexget/plugins/daemon/irc.py

```diff
@@ -667,6 +673,13 @@ def irc_update_config(manager):
         stop_irc(manager)
         return
 
+    if irc_bot is None:
+        log.error('ImportError: irc_bot module not found. Shutting down daemon.')
+        stop_irc(manager)
+        from flexget.manager import manager
```

Comment:
- Imports should be at the top of the module unless some very good reason
- Manager shutdown seems rather drastic, I don't think any plugin does that. Simply disable the plugin instead?